src/__experimental__/components/select/select.style.js

```diff
@@ -0,0 +1,34 @@
+import styled, { css } from 'styled-components';
+import PropTypes from 'prop-types';
+import baseTheme from '../../../style/themes/base';
+import { isClassic } from '../../../utils/helpers/style-helper';
+
+const StyledSelectPillContainer = styled.div`
```

Comment:
couldn't it be styled(Pill)?

Reply:
The updated Pill component has not been merged yet so I had to go with this for now

Reply:
I have updated it to use the new `StyledPill` https://github.com/Sage/carbon/pull/2163/commits/f3a8ce2624caf0670a23ae6ee8dca5c0dddfeed3